content/configuration/external-configs.md

```diff
@@ -4,10 +4,36 @@ sort: 3
 contributors:
   - sokra
   - skipjack
+  - tarang9211
 ---
 
-...
+Webpack lets you define you configuration files in any language. The list of supported file extensions can be found at the [node-interpret](https://www.npmjs.com/package/interpret) package. Webpack will still interpret your file as pure JS!
```

Comment:
How is "Webpack will still interpret your file as pure JS" meant? interpret preprocesses/transpiles the file, and gives the result to webpack?